docs/preview/features/ps/keyvault.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Scripts related to interacting with Azure Key Vault"
+layout: default
+---
+
+# Azure Key Vault scripting
+
+## Installation
+
+To have access to the following features, you have to import the module:
+
+```powershell
+PS> Import-Module -Name Arcus.Scripting.PS.KeyVault
+```
+
+## Get all access policies of a key vault
+
+Lists the current available access policies of the Azure key vault resource into a new variable availble at runtime during a Azure DevOps build.
```

Comment:
small typo: 
- availble -> available
- a Azure DevOps -> an Azure DevOps

Reply:
Agreed, extended it in a proposal below to not only pin it to Azure DevOps.

Reply:
Added proposal below.